opossum-tool/OpossumUI
# Document search feature in locator

### Summary of changes

Updates the USER_GUIDE.md with the updates to the locator introduced on this branch.

### Context and reason for change

Keep docs up to date

### How can the changes be tested

See the [updated user guide](https://github.com/MarkusObendrauf/OpossumUI/blob/document-search/USER_GUIDE.md#locator)
